src/pages/jj/Main/MainComponentMap/MainComponent.js

```diff
@@ -0,0 +1,162 @@
+import React, { Component } from 'react';
+import CommentList from './CommentListMap/CommentListMap';
+
+import uuid from 'react-uuid';
+
+class MainComponent extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      list: [],
+      comment: '',
+      heartColor: false,
+      className: [],
+      bigHeart: false,
+    };
+  }
+
+  // #부모요소에서 받아온 APIKEY값의 comment값
+  componentDidMount() {
+    this.setState({ list: this.props.feedList.comment });
+  }
+
+  // form submit시 함수 실행(댓글 생성)
+  handleSubmit = e => {
+    const { list, className, comment } = this.state;
+    e.preventDefault();
+
+    if (this.state.comment) {
+      const newList = {
+        id: uuid(),
+        comment: comment,
+        class: false,
+      };
+      this.setState({
+        list: [...list, newList],
+        className: [...className, newList.class],
+      });
+      this.setState({ comment: '' });
+    }
+  };
+
+  // 각 댓글에 좋아요 버튼 활성화 함수
+  changeHeartColor = (elId, index) => {
+    const { className } = this.state;
+    let copy = [...className];
+    copy[index] = !copy[index];
+    this.setState({ className: copy });
+  };
+
+  // #인풋 요소를 comment 값으로 넣어주는 함수
+  inputOnChange = e => {
+    this.setState({ comment: e.target.value });
+  };
+
+  // #삭제함수
+  deleteBtn = (elId, index) => {
+    const { list, className } = this.state;
+    this.setState({
+      list: list.filter(el => {
+        return el.id !== elId;
+      }),
+      className: className.filter((el, i) => {
+        return i !== index;
+      }),
+    });
+  };
+
+  clickBigHeart = () => {
+    const { bigHeart } = this.state;
+
+    setTimeout(() => {
+      this.setState({ bigHeart: !bigHeart });
+    }, 1000);
+  };
+
+  render() {
+    const { feedList } = this.props;
+    const { list, comment, className, bigHeart } = this.state;
+    return (
+      <>
+        <div className="mainBg">
+          <section className="mainTop">
+            <div className="mTopProfile">
+              <img
+                alt="account"
+                src={feedList.userImg}
+                className="mTopAccount"
+              />
+              <p className="mTopUserName">{feedList.userName}</p>
+            </div>
+            <p className="mTopThreeDot">…</p>
+          </section>
+          <section className="mainImg">
+            {bigHeart ? <i className="fas fa-heart bigHeart" /> : null}
```

Comment:
```suggestion
            {bigHeart && <i className="fas fa-heart bigHeart" />}
```
true 케이스만 렌더할 경우 &&를 사용해줄 수 있습니다